src/app.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// Write your code here
+import readline from 'readline';
+import { generateRandomNumber } from './modules/generateRandomNumber';
+import { checkIsValidUserInput } from './modules/checkIsValidUserInput';
+import { getBullsAndCows } from './modules/getBullsAndCows';
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+const numberToGuess = generateRandomNumber();
+
+terminal.question('Guess the number', (userNumber) => {
+  if (!checkIsValidUserInput(userNumber)) {
+    return 'Input is not a valid 4-digit number';
```

Comment:
The return statement here will not send the message to the user. Consider using console.log to display the message to the user.